### ISSUE TITLE
Copy as default instead of move

### ISSUE THREAD

**Issue by jaafonso** [RealRaven2000/QuickFolders#363]
I have a Exchange/Outlook account at work. I keep the important messages inside local folders and delete the others.

When I move a message to a local folder, it is removed from the Outlook server, but I'd like to keep a copy on the server.

The only solution that I found was:

1. Press CTRL.
2. Copy the message to the local folder.
3. Delete the message from the Inbox.

I'd like to avoid the first step, that is, have copy as default and press CTRL (or other key to move).

Regards,

jaafonso

**Comment by RealRaven2000** [comment]
For the moment, you can use the modified key CTRL while dragging emails to force a copy of the message. Having a global setting for copying may make sense but it sounds like it would be more useful if it was a per-server setting?

**Comment by jaafonso** [comment]
Yes, I'm using the CTRL key to copy (step 1), but I'd like to remove this step. A per-server setting may be more useful for others, but for all my servers (including Gmail) I generally need to copy instead of moving the messages to my Thunderbird local folders.

**Comment by jaafonso** [comment]
I created a rule in Outlook to make a copy of all received messages to other folder, so now I can move the message from the Inbox to a local Thunderbird folder and they remain in the server. It is a better solution, so for me the issue is solved.

**Comment by RealRaven2000** [comment]
> I created a rule in Outlook to make a copy of all received messages to other folder, so now I can move the message from the Inbox to a local Thunderbird folder and they remain in the server. It is a better solution, so for me the issue is solved.

OK, we might keep this one open for a while. it may come up again as a request... Not sure if it would benefit a lot of users, so it can be discussed here first.
